core/src/main/java/com/microsoft/applicationinsights/InstrumentationKey.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.applicationinsights;
```

Comment:
All the new classes are missing copyrights.

Reply:
👍 I'll add in separate PR  (#1678 is based on this PR already so don't want to cause it problems)